packer/payload/prepare-ami.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash -eux
+# Copyright 2017 by the contributors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+SOURCE_DIR="$(cd "$(dirname "$0")"; pwd)"
+KUBERNETES_RELEASE="v1.5.4"
+KUBEADM_RELEASE="v1.6.0-alpha.0.2074+a092d8e0f95f52"
+CNI_RELEASE="07a8a28637e97b22eb8dfe710eeae1344f69d16e"
+
+## Make sure we get the latest updates since the base image was released
+apt-get update -q
+apt-get upgrade -qy
+
+## TODO: Update docker to use overlay2 by default
+apt-get install -qy docker.io jq python-pip python-setuptools ebtables socat ntp
```

Comment:
We might want to install docker from their repos instead of whatever ubuntu is shipping. That is, at least, the preferred method from Docker.

Also note that 1.12.x is the latest to be qualified by the SIG-node team so we should stick with that.

Code snippet for installing that: https://gist.github.com/jbeda/60db714598f6c6a7a0558086e2225ab4